build/meshmodel_metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+	"primary-color":   "#291953",
+	"secondary-color": "#6942c9",
+	"shape":           "circle",
+	"logo-URL":        "https://github.com/cncf/artwork/blob/master/projects/kuma/icon/white/kuma-icon-white.svg",
```

Comment:
@Revolyssup Upon registration of these components, does this URL change? It's not useful otherwise.

Reply:
Okay I will drop this. This is present in the spreadsheet so I propagated it, just in case.